libs/commons/Logs_helpers.ml

```diff
@@ -1,34 +1,35 @@
-(* ANSI escape sequences for colored output, depending on log level *)
```

Comment:
you'll need to also keep this code, otherwise we get regressions in make osempass